frontend/device/kobo/nickel_conf.lua

```diff
@@ -53,9 +53,7 @@ function NickelConf.frontLightLevel.get()
         -- FrontLightState config, so don't normalize the value here yet.
         return tonumber(new_intensity)
     else
-        -- In NickelConfSpec, require("device") won't return KoboDevice
-        local powerd = require("device/kobo/powerd")
-        local fallback_fl_level = powerd.fl_intensity or 1
+        local fallback_fl_level = 20
```

Comment:
This "environment brightness" function is very hardware dependent, for KOReader, now, that means the Voyage & some Android devices.  It would be nice, but I don't see how any Kobo/Nickel device would be detecting the environmental brightness.  With the interest in keeping the power consumption as low as possible, the idea of an "Auto Frontlight", at any level setting, is troubling.  Putting a number of status checks (on powerd, included) into the reader loop logic, can make for unnecessary vulnerabilities, as well as potential performance affects. 
It might be an example of something that should be left to the underlying OS/firmware of the device, where it can be run in its own thread.  (You asked for thoughts, sorry.)